src/services/api.ts

```diff
@@ -87,6 +92,20 @@ export class YdbEmbeddedAPI extends AxiosWrapper {
             return config;
         });
 
+        // Add traceId to response if it exists
+        this._axios.interceptors.response.use(function (response) {
+            if (response.data && response.headers['traceresponse']) {
+                const traceId = response.headers['traceresponse'].split('-')[1];
+
+                response.data = {
+                    ...response.data,
+                    meta: {...response.data.meta, traceId},
+                };
```

Comment:
I recently added interceptor, that allows to enable tracing for all requests (you can find it above). So this code now relates to all requests. So,
1. Check response type to ensure it's an object. Now it creates objects for `null` responses and converts arrays to objects
2. It's better to rename `meta` to something like `_meta` to reduce probability of similar naming and point out that it's about request and doesn't relate to data
3. It will be good to add some general wrapper type for responses, not to add meta to all needed responses, but to use it like `ResponseWrapper<ResponseType>`
```
type ResponseWrapper<T> = T extends Array<any> ? T : T extends Object ? T & {_meta: {traceId: string}} : T;
```

Reply:
made 1 and 2